src/test/resources/data/workflows.csv

```diff
@@ -0,0 +1,3 @@
+workflowId,workflowVersion,descriptorType
+#workflow/github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM,3.0.0,WDL
```

Comment:
looks like a comment, but comments aren't supported in csv so maybe it's ok to leave as is

Reply:
I renamed the column header to trsId, to make it clearer that the `#` is intentional (it's not a comment).

e5e32a57147b623431baaac560ce837c1ea3df21